README.md

```diff
@@ -239,6 +239,8 @@ This step fetches the specified URL, without the session and cookies of the curr
 
 ## Changelog
 
+3.3.1: pause when committing and submitting, to allow asynchronous page content save operations to complete. Increases test stability.
```

Comment:
This can be included in 3.3.0 since it hasn't been published. I think we should also date the version here to help with this kind of situation.